src/main/com/cqfn/degitx/simulator/data/GraphData.kt

```diff
@@ -0,0 +1,39 @@
+package com.cqfn.degitx.simulator.data
+
+/**
+ * Data class to store, load and save graph structure.
+ */
+data class GraphData(
+    val graph: Graph
+)
+
+data class Graph(
+    val edges: List<Edge>,
+    val nodes: List<Node>
+)
+
+data class Edge(
+    val head: String,
+    val tail: String
+)
+
+data class Node(
+    val hw: Hw,
+    val id: String,
+    val state: String
+)
+
+data class Hw(
```

Comment:
Please rename it to Hardware, It takes a couple of seconds to understand what it is.

Reply:
Agree. Thank you, @stepanvalyavskiy , it is fixed.